### DISCUSSION TITLE
Lag Issues with OpenBoard on macOS (Version 1.7.2)

### DISCUSSION THREAD

**Discussion by VivekM8737** [OpenBoard-org/OpenBoard#1192] [Q&A]
Hello OpenBoard Community,

I have been using OpenBoard on Windows for a long time and have never faced any issues. However, after switching to macOS, I am experiencing significant lag with the application. Currently, I am using version 1.7.2, and the lag is making it difficult to use the software efficiently.

Has anyone encountered similar performance issues on macOS? I would appreciate any suggestions or solutions to resolve this problem and improve the overall experience on macOS.

**Comment by bruening-bw** [comment]
There are delay problems in OpenBoard under all operating systems. I use Linux and can provoke the input lag by changing the pen settings.

![grafik](https://github.com/user-attachments/assets/d8e773b4-5915-49e7-b2df-d2b90e3b749c)

If I change the size of the preview circle to a value less than 5px, there are delays. If I reset the value to 5px, everything is fine. While playing with the settings, I found out that changing the size to a value higher than 5px doesn't change anything anymore. Whether I set 5px or 50px makes no difference. Disabling it also leaves the default behavior. For values below 5px there is input lag. But even here it makes no difference whether I have set 4px or 1px. I think this setting only allows two states:

1. small cursor with input lag (<5px)
2. normal cursor without input lag (>4px or deactivated)

Another way to provoke input lag is to disable pressure sensitivity.

I believe that most lag problems are somehow related to these settings. At least it is a clue for the developers where to look for the cause of the problems.
